global.json

```diff
@@ -0,0 +1,6 @@
+{
+  "sdk": {
+    "version": "6.0.0",
```

Comment:
Fable 3 only works on 6.x.  We could goto 4 but I don't want to go for touching more than I need to.